csharp_dotnetcore/21.Custom-Dialogs/MainBot.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+
+namespace Custom_Dialogs
+{
+    /// <summary>
+    /// This bot illustrates a simple prompt.
```

Comment:
I would lean towards this comment should be more robust

Reply:
yes done